third-party/linshare/src/test/java/org/apache/james/linshare/LinshareTest.java

```diff
@@ -67,12 +67,12 @@ void linshareShouldStart() {
     }
 
     @Test
-    void linshareShouldShouldHaveATechnicalAccountConfigured() {
+    void linshareShouldHaveATechnicalAccountConfigured() {
         List<TechnicalAccountDTO> technicalAccounts = linshareExtension.getAllTechnicalAccounts(LinshareFixture.ADMIN_ACCOUNT);
 
         assertThat(technicalAccounts).anySatisfy(account -> SoftAssertions.assertSoftly(softly -> {
-            softly.assertThat(TECHNICAL_ACCOUNT.getUsername()).isEqualTo(account.getName());
-            softly.assertThat(TECHNICAL_PERMISSIONS).containsExactlyElementsOf(account.getPermissions());
+            softly.assertThat(account.getName()).isEqualTo(TECHNICAL_ACCOUNT.getUsername());
+            softly.assertThat(account.getPermissions()).containsExactlyElementsOf(TECHNICAL_PERMISSIONS);
```

Comment:
IMO containsOnlyElementsOf should be enough, we don't care about the order.

Reply:
The TechnicalAccount should have permissions we grant to it by code at test.So it should ```containsExactlyElementsOf``` which we granted.Should you agree with it?